src/etc/inc/pfsense-utils.inc

```diff
@@ -1453,7 +1456,7 @@ function convert_seconds_to_dhms($sec, $showhoursonly = false) {
 					(int)(($sec % 3600)/60),
 					$sec % 60
 				);
-	return ($d > 0 ? $d . 'd ' : '') . sprintf('%02d:%02d:%02d', $h, $m, $s);
+	return ($d > 0 ? $d . 'd ' : '') . sprintf(($showHMS ? '%02dh %02dm %02ds' : '%02d:%02d:%02d'), $h, $m, $s);
```

Comment:
For these one big line statements are a bit confusing and hard to read. Do you mind to rework it using variables and whatever is necessary to reduce these ternary operators?